transcom/mymove
# B 18186 remove ppm completer

B-18186

## Summary

### How To Test
Because this story focuses on removing unused code. It is unlikely it should affect currently active code. However the closest means to do so would be to:
1. Ensure that all Jest/Playwright/Go tests pass (or that no new tests are failing).
2. Do a brief run through with all activities relating to creation, approval,deny, delete, update operations on PPMs.
